outputs.tf

```diff
@@ -8,6 +8,11 @@ output "function_name" {
   value       = "${element(concat(aws_lambda_function.lambda.*.function_name, aws_lambda_function.lambda_with_dl.*.function_name, aws_lambda_function.lambda_with_vpc.*.function_name, aws_lambda_function.lambda_with_dl_and_vpc.*.function_name), 0)}"
 }
 
+output "invoke_arn" {
+  description = "The ARN to be used for invoking the Lambda Function from API Gateway"
```

Comment:
Please change `Function -> function`. I know AWS refers to them with and without a capital `F` in different places but the other outputs use lowercase so let's go with that. Otherwise, it looks good. Thanks.